app/assets/javascripts/discourse/ember-cli-build.js

```diff
@@ -20,6 +21,13 @@ module.exports = function (defaults) {
     },
   });
 
+  // Ember CLI does this by default for the app tree, but for our extra bundles we
+  // need to do it ourselves in production mode.
+  const isProduction = EmberApp.env().indexOf("production") !== -1;
```

Comment:
```suggestion
  const isProduction = EmberApp.env().includes("production");
```

Or could `===` be enough there? are there any production-like values?

Reply:
Yeah annoyingly it's a string like `environment: production`